src/libraries/illink-sharedframework.targets

```diff
@@ -66,6 +56,49 @@
+      <ProjectILLinkSuppressionsWasmSpecificFile>$(ProjectILLinkSuppressionsFile).wasm</ProjectILLinkSuppressionsWasmSpecificFile>
+      <CoreLibSharedILLinkSuppressionsFilesPrefix>$(CoreLibSharedDir)ILLink\ILLink.Suppressions.Shared</CoreLibSharedILLinkSuppressionsFilesPrefix>
+    </PropertyGroup>
+
+    <!-- Collate XML files not bin-placed in earlier per-library linker run -->
+    <ItemGroup>
+      <!-- CoreLib doesn't use bin-place logic -->
+      <_SuppressionsXmls Include="$(CoreLibSharedILLinkSuppressionsFilesPrefix).xml" />
+      <_SuppressionsXmls Condition="'$(TargetOs)' != 'Windows_NT'" Include="$(CoreLibSharedILLinkSuppressionsFilesPrefix).NonWindows.xml" />
+      <_SuppressionsXmls Condition="'$(RuntimeFlavor)' == 'Mono'" Include="$(MonoProjectRoot)netcore\System.Private.CoreLib\$(ProjectILLinkSuppressionsFile).xml" />
+
+      <!-- Assemblies that were not included in per-library linker run -->
+      <_SuppressionsXmls Include="$(LibrariesProjectRoot)Microsoft.CSharp\$(ProjectILLinkSuppressionsFile).xml" />
+      <_SuppressionsXmls Include="$(LibrariesProjectRoot)Microsoft.VisualBasic.Core\$(ProjectILLinkSuppressionsFile).xml" />
+      <_SuppressionsXmls Include="$(LibrariesProjectRoot)System.Drawing.Primitives\$(ProjectILLinkSuppressionsFile).xml" />
+      <_SuppressionsXmls Include="$(LibrariesProjectRoot)System.Linq.Queryable\$(ProjectILLinkSuppressionsFile).xml" />
+      <_SuppressionsXmls Include="$(LibrariesProjectRoot)System.Net.Http\$(ProjectILLinkSuppressionsFile).xml" />
+      <_SuppressionsXmls Include="$(LibrariesProjectRoot)System.Runtime.Serialization.Formatters\$(ProjectILLinkSuppressionsFile).xml" />
+      <_SuppressionsXmls Include="$(LibrariesProjectRoot)System.Security.Cryptography.Algorithms\$(ProjectILLinkSuppressionsFile).xml" />
+      <_SuppressionsXmls Include="$(LibrariesProjectRoot)System.Security.Cryptography.Cng\$(ProjectILLinkSuppressionsFile).xml" />
+      <_SuppressionsXmls Include="$(LibrariesProjectRoot)System.Security.Cryptography.Csp\$(ProjectILLinkSuppressionsFile).xml" />
+      <_SuppressionsXmls Include="$(LibrariesProjectRoot)System.Security.Cryptography.X509Certificates\$(ProjectILLinkSuppressionsFile).xml" />
+      <_SuppressionsXmls Include="$(LibrariesProjectRoot)System.Security.Cryptography.X509Certificates\$(ProjectILLinkSuppressionsMonoSpecificFile).xml" />
+
+      <_SuppressionsXmls Condition="'$(TargetArchitecture)' == 'wasm'"
+                         Include="$(LibrariesProjectRoot)System.Private.Runtime.InteropServices.JavaScript\$(ProjectILLinkSuppressionsWasmSpecificFile).xml" />
+
+      <!-- TODO: bin-place the above suppressions files to the ILLinkTrimAssemblySuppressionsXmlsDir directory below for easier debugging -->
+    </ItemGroup>
+
+    <ItemGroup>
+      <_SuppressionsXmls Include="$(ILLinkTrimAssemblySuppressionsXmlsDir)*.xml" />
```

Comment:
>ILLinkTrimAssemblySuppressionsXmlsDir [](start = 36, length = 37)

oh disregard my comment of moving the properties into illink.targets, now I see that you use them here.